homeassistant/components/ring/strings.json

```diff
@@ -6,26 +6,39 @@
         "data": {
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "username": "Your Ring cloud username.",
+          "password": "Your Ring cloud password."
```

Comment:
I believe it's "Ring account", not "Ring cloud". Let's be consistent in the strings and also match what Ring calls it on their website.